### DISCUSSION TITLE
Cut out the center of image

### DISCUSSION THREAD

**Discussion by fgiacosa** [ImageMagick/ImageMagick#4582] [Help] [answered]
Hi
I would like to cut off the center of an image leaving only the edges. Is it possible to do this with ImageMagick? I tried to search but found nothing. Here is an example of the result.
Thank you

Version: ImageMagick 7.1.0-14 Q16-HDRI arm 2021-11-14 https://imagemagick.org

![img-cutout](https://user-images.githubusercontent.com/6873426/145695179-bc05ba82-76bb-4370-8e76-7bafad054188.png)
![img](https://user-images.githubusercontent.com/6873426/145695181-219d4369-20a2-4581-830d-361ccf323103.jpg)

**Comment by fmw42** [comment, marked as answer]
If you want that space to be transparent, your have to save as PNG not JPG.  If you want it to be white, you can save as JPG. Here is how to make it transparent in IM 7 Unix syntax.

```
magick 145695181-219d4369-20a2-4581-830d-361ccf323103.jpg \
\( +clone -fill white -colorize 100 -fill black -draw "rectangle %[fx:.1*w],%[fx:.1*h] %[fx:.9*w],%[fx:.9*h]" +write xx.png \) \
-alpha off -compose copy_opacity -composite \
result.png

```

![result](https://user-images.githubusercontent.com/13458218/145696008-680e4a48-6fcc-4e2d-a20c-b7a31b55f72e.png)

If using IM 6, then you will have to specify the rectangle coordinates directly. If using Windows, remove the \ from the parentheses and replace the end of line \ with ^

**Reply by fgiacosa** [reply]
Thank you very much!! 😄
